client/a8c-for-agencies/sections/marketplace/hosting-overview/index.tsx

```diff
@@ -26,8 +30,17 @@ export default function Hosting() {
 					<Title>{ translate( 'Marketplace' ) }</Title>
 
 					<Actions>
-						{ /* FIXME: We will need to have shared state for shopping cart items */ }
-						<ShoppingCart items={ [] } onRemoveItem={ () => {} } onCheckout={ () => {} } />
+						<ShoppingCart
+							items={ selectedItems }
+							onRemoveItem={ ( item: ShoppingCartItem ) => {
+								setSelectedItems(
+									selectedItems.filter( ( selectedItem ) => selectedItem !== item )
+								);
+							} }
```

Comment:
I think we can move the remove item logic to the useShoppingCart hook

Reply:
Nice suggestion. This has been applied 👍